examples/with-redux/lib/redux/slices/counterSlice/fetchIdentityCount.ts

```diff
@@ -1,7 +1,7 @@
 export const fetchIdentityCount = async (
   amount = 1
 ): Promise<{ data: number }> => {
-  const response = await fetch('http://localhost:3000/api/identity-count', {
+  const response = await fetch(`http://${$window.location.hostname}:3000/api/identity-count`, {
```

Comment:
```suggestion
  const response = await fetch(`/api/identity-count`, {
```

Ig this should also work. I'll also use proper port and hostname